killswitch.py

```diff
@@ -0,0 +1,189 @@
+"""Fetch kill switches from EDMC Repo."""
+from typing import Dict, List, NamedTuple, Optional, Union, cast
+
+import requests
+import semantic_version
+
+import config
+import EDMCLogging
+
+logger = EDMCLogging.get_main_logger()
+
+# DEFAULT_KILLSWITCH_URL = 'https://github.com/EDCD/EDMarketConnector'
+DEFAULT_KILLSWITCH_URL = 'http://127.0.0.1:8080/killswitches.json'
```

Comment:
We need to have a think about where to host the kill switches.  Given the appcast file is via raw github file I see no reason not to utilise the same for this file.  

However, what branch ?  The appcast is looked for in the `releases` branch.  Given the version/branch-agnostic nature of the killswitch file (pertinent application versioning is within it) it could also be in the `releases` branch.  We would just need to remember to take care with merges so as to not accidentally unintentionally change the appcast file.  In fact I think the defined procedure should be to simply commit only to the `releases` branch for edits to the killswitch file.

Reply:
Could be in releases or in main, I've left it because Im not sure where exactly as of yet. Releases sounds like the better option as its where appcast stuff is done already, and having that all together seems like a better idea